### DISCUSSION TITLE
Namecheap

### DISCUSSION THREAD

**Discussion by leedsrhinos** [boxbilling/boxbilling#1218] [General]
evening all is there any way that we can get namecheapas a domain registra via boxbilling?
if so how do we do this
is there any extension etc
Thanks

**Comment by timothygwebb** [comment]
@leedsrhinos Please review the following website it seems that Namecheap offers through cPanel and softaculous a boxbilling adapter https://www.namecheap.com/support/knowledgebase/article.aspx/1063/22/what-scripts-does-softaculous-in-cpanel-offer. The boxbilling team has not reviewed this or endorse this.  I have also found another boxbilling adapter that has been developed for Namecheap. The boxbilling team has not reviewed or endorse the following adapter located at https://github.com/Eptikar-IT-Solutions/boxbilling-namecheap/blob/master/bb-library/Registrar/Adapter/Namecheap.php
